docs/python-reference/linear-algebra.ipynb

```diff
@@ -93,7 +93,7 @@
    "id": "60e1b618-ec63-4da2-8374-a4655fdb0a66",
    "metadata": {},
    "source": [
-    "This works in an identical manner for matrices with the properties `x11`, `x12`, `x13`, `x21`, `x22`, `x23`, `x31`, `x32`, and `x33`.\n",
+    "This works in an identical manner for matrices with the properties `xx`, `xy`, `xz`, `yx`, `yy`, `yz`, `zx`, `zy`, and `zz`.\n",
```

Comment:
So what is the plan if the matrix has more than 3 rows or columns?

Reply:
In general, the idea is to support arbitrary custom structured types, so the property names would be given by the user.

It is not entirely clear if and how we would support matrices with different shapes, since this would also require actual support for operations.